WORKSPACE

```diff
@@ -250,3 +250,12 @@ scala_maven_import_external(
     licenses = ["notice"],
     server_urls = MAVEN_SERVER_URLS,
 )
+
+maven_jar(
+    name = "com_illicitonion_scalac_composite_reporter",
+    artifact = scala_mvn_artifact(
+        "com.illicitonion:scalac-composite-reporter:0.0.1",
```

Comment:
You'll probably want to put the code for this somewhere in this repo, and make sure it's built locally. There's a fun bootstrapping problem in that it _needs _ to be scala (it _should_ be able to be java, but I couldn't get a pure-java reporter to work for weird reasons related to how scalac specifically handles matching against case classes IIRC), but it's depended on by the scala compilation rules themselves... But yeah, this probably shouldn't rely on a random binary artifact :)

Reply:
big +1. Probably the bit that concerns me the most in this review is depending on this artifact

Reply:
Was able to implement it in java, done!